unity-client/Assets/Scripts/MainScripts/DCL/Bridges/DebugBridge.cs

```diff
@@ -52,5 +53,31 @@ public void SetDisableAssetBundles()
         {
             RendereableAssetLoadHelper.loadingType = RendereableAssetLoadHelper.LoadingType.GLTF_ONLY;
         }
+
+        public void DumpRendererLockersInfo()
+        {
+            bool prevLogValue = Debug.unityLogger.logEnabled;
+            Debug.unityLogger.logEnabled = true;
+
+            RenderingController renderingController = FindObjectOfType<RenderingController>();
+            if (renderingController == null)
+            {
+                Debug.Log("RenderingController not found. Aborting.");
+                return;
+            }
+
+            Debug.Log($"Renderer is locked? {!renderingController.renderingActivatedAckLock.isUnlocked}");
+            Debug.Log($"Renderer is active? {CommonScriptableObjects.rendererState.Get()}");
+
+            System.Collections.Generic.HashSet<object> lockIds =
+                renderingController.renderingActivatedAckLock.GetLockIdsCopy();
+
+            foreach (var lockId in lockIds)
+            {
+                Debug.Log($"Renderer is locked by id: {lockId} of type {lockId.GetType()}");
```

Comment:
Maybe we can find a way to have a descriptive string for the locks?

A few ways to do this:
- The locks can work with an interface with `string GetLockDescription()` method instead of an `object`.
- change `AddLock` signature to receive an optional `description` string and change the internals to be able to get that description.

Reply:
I was about to use the type of the locker and cast it to get more info... but right now we don't have that much lockers, so I think it doesn't worth the effort right now... and most of the` lockId.ToString() ` are very descriptives, specially scenes

Reply:
Fair enough